include/amqpcpp/callbacks.h

```diff
@@ -45,7 +45,7 @@ using DeleteCallback        =   std::function<void(uint32_t deletedmessages)>;
  *  When retrieving the size of a queue in some way
  */
 using EmptyCallback         =   std::function<void()>;
-using SizeCallback          =   std::function<void(uint32_t messagecount)>;
+using SizeCallback          =   std::function<void(uint64_t messagecount)>;
```

Comment:
I'm wondering whether it would be worth it to add a CountCallback that retains the current signature and use that as the type of `_countCallback` in DeferredGet which is returning a value that is uint32_t by protocol.  Most other uses of `SizeCallback` are providing message size which does need to be uint64_t.